incubator/java-microprofile/stack.yaml

```diff
@@ -18,3 +18,4 @@ requirements:
     appsody-version: ">= 0.5.0"
 templating-data:
     libertyversion: '19.0.0.12'
+deprecated: 04/14/2020
```

Comment:
Would you be able to add some info text to the `deprecated` element stating why the stack was deprecated, as you have done in the description and the readme. The value of the `deprecated` element is displayed when an appsody init | run | test | debug | deploy | extract are executed, so having some additional info other than date might be useful.

Reply:
We have requested the format to be used to be "<date> - <deprecation reason>"

Reply:
Thanks @groeges - changes made.